angular/projects/public-nrpti/src/assets/styles/themes/default.scss

```diff
@@ -1,5 +1,5 @@
 // Main Colors
-$primary:                       #036;
+$primary:                       #38598a;
```

Comment:
I don't think this is the standard colour we should be using for $primary, #38598a is only for button colour in the designs.